src/Sarif.Viewer.VisualStudio.Core/Models/SarifErrorListItem.cs

```diff
@@ -94,8 +94,8 @@ public SarifErrorListItem(Run run, int runIndex, Result result, string logFilePa
             this.WorkingDirectory = Path.Combine(Path.GetTempPath(), this.RunIndex.ToString());
             this.HelpLink = this.Rule?.HelpUri;
 
-            this.RawMessage = result.GetMessageText(rule);
-            this.ShortMessage = this.RawMessage;
+            this.RawMessage = result.GetMessageText(rule, concise: false).Trim();
```

Comment:
>Trim() [](http://example.com/codeflow?start=74&length=6)

open question, if the trim is needed

Reply:
This is for clean data, in the event that, for example, such characters cause some undesirable behavior in a future VS update